### ISSUE TITLE
Wrong text in 'Update delegation' -> Passive

### ISSUE THREAD

**Issue by EmilLa1** [Concordium/concordium-reference-wallet-ios#286]
**Bug Description**
The text does not reflect what Figma shows. 

**Steps to Reproduce**

Update delegation from a pool to the 'Passive pool'.

**Expected Result**
Text is 
```
Passive delegation is an alternative to delegation to a specific baker pool that has lower rewards. With passive delegation you do not have to worry about the uptime or quality of a baker node. For more info you can visit developer.concordium.software
```

**Actual Result**
But the text is different.

**Versions**
iOS 3.0.0 (48)

**Comment by Niljak** [comment]
Fixed in version 3.0.0 (49)

**Comment by EmilLa1** [comment]
Looks good. 
3.0.0 (49)
